### ISSUE TITLE
During creation of primitive, the "CTRL+LMB click" step is redundant, and should be removed.

### ISSUE THREAD

**Issue by raindropsfromsky** [jayanam/fast-carve#47]
The "CTRL+LMB click" step is redundant, and should be removed.

Here is an example of how I use the addon:

My settings are shown below:
![image](https://user-images.githubusercontent.com/9047168/62825866-f2f0c300-bbcf-11e9-9511-638f50e27c33.png)

And here is the moment when I finished extruding the polygon.
![image](https://user-images.githubusercontent.com/9047168/62825859-df455c80-bbcf-11e9-9413-60023903c9a4.png)

As soon as I finish the extrusion (by releasing the LMB), the addon should create the object based on my preferences (shown in the first figure, above.) There is no need to have a separate "CTRL+LMB click" step.

This is how the new **Blueprint** tool works in the new sculpt branch (by Pablo Dobarro), and it is much faster.

**Comment by jayanam** [comment]
I agree that Ctrl + Left click is redundant, it can be left click. But extruding with holding the mouse button down and applying when releasing is not a good idea. It is better to start an extrusion - like it is - and finish it with an explicit click - because otherwise you cant for example rotate the view with middle mouse button pressed while extruding.

**Comment by raindropsfromsky** [comment]
Yes, agree. But then do we need to really rotate the view for a plain extrusion operation?

In any case I have suggested a fast draw mode (#51), where a shape can be created with just three clicks. If that mode is available, the user can choose between this cautious approach and a rapid-fire approach.
